package.json

```diff
@@ -109,7 +109,7 @@
     "gulp-sourcemaps": "^2.6.4",
     "gulp-uglify": "^2.0.1",
     "istanbul": "^0.4.5",
-    "mocha": "^3.2.0",
+    "mocha": "^6.2.1",
```

Comment:
@tofixx are you sure that you wan't v6? The Pull title and branch name says v5.
If this is what you wan't, feel free to merge.